test/blackbox-tests/test-cases/output-obj/run.t

```diff
@@ -12,11 +12,11 @@
       ocamlopt test$ext_dll
 
   $ dune build @runtest
+       dynamic alias runtest
+  OK: ./dynamic.exe ./test$ext_dll
         static alias runtest
   OK: ./static.bc
-       dynamic alias runtest
-  OK: ./dynamic.exe ./test.bc.so
         static alias runtest
   OK: ./static.exe
        dynamic alias runtest
-  OK: ./dynamic.exe ./test$ext_dll
+  OK: ./dynamic.exe ./test.bc.so
```

Comment:
Surprised that this tests changed. Is it b/c the regex is fragile? In any case, I don't believe in making the output portable via post-processing anymore so I'm not concerned about this.